sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -1,5 +1,73 @@
 namespace Azure.AI.Personalizer
 {
+    public partial class DecisionContext
```

Comment:
Should the new public types also have the prefix `Personalizer` to help users understand their context and provide consistency with the existing types in this library?

Reply:
Every class under Azure.AI.Personalizer is for Personalizer. I would say the Personalizer prefix is unnecessary.